src/logging.cpp

```diff
@@ -286,6 +312,8 @@ std::string LogCategoryToStr(BCLog::LogFlags category)
         return "txreconciliation";
     case BCLog::LogFlags::SCAN:
         return "scan";
+    // Add new entries before this line.
+
```

Comment:
nit: reorder BCLog::LogFlags::NONE here in line with the definition in `logging.h`?